Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -2356,6 +2377,12 @@ def gen2018HiMix(fragment,howMuch):
 steps['RECOPRMXUP18_PU25']=merge([
         {'--conditions':'auto:phase1_2018_realistic','--era':'Run2_2018','--procModifiers':'premix_stage2'},
         step3Up2015Defaults])
+steps['RECOPRMXUP18_PU25_RD_test']=merge([
+        {'--conditions':'auto:phase1_2018_realistic','--era':'Run2_2018','--procModifiers':'premix_stage2', '--customise_commands':"\"process.EcalLaserCorrectionServiceMC = cms.ESProducer('EcalLaserCorrectionServiceMC') \\n process.GlobalTag.toGet = cms.VPSet( cms.PSet(     record = cms.string('EcalLaserAPDPNRatiosMCRcd'),  tag = cms.string('EcalLaserAPDPNRatios_Run_Dep_MC_first_IOV'),  connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS') ), cms.PSet(     record = cms.string('EcalLaserAPDPNRatiosRcd'),  tag = cms.string('EcalLaserAPDPNRatios_Run_Dep_MC'),  connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS') ) )  \\n process.ecal_sim_parameter_map.timeDependent=cms.bool(True) \""},
+        step3Up2015Defaults])
+steps['RECOPRMXUP18_PU25_RD']=merge([
+        {'--conditions':'auto:phase1_2018_realistic','--era':'Run2_2018','--procModifiers':'premix_stage2', '--customise_commands':"\"process.EcalLaserCorrectionServiceMC = cms.ESProducer('EcalLaserCorrectionServiceMC') \\n process.GlobalTag.toGet = cms.VPSet( cms.PSet(     record = cms.string('EcalLaserAPDPNRatiosMCRcd'),  tag = cms.string('EcalLaserAPDPNRatios_Run_Dep_MC_first_IOV'),  connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS') ), cms.PSet(     record = cms.string('EcalLaserAPDPNRatiosRcd'),  tag = cms.string('EcalLaserAPDPNRatios_Run_Dep_MC'),  connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS') ) )  \\n process.ecal_sim_parameter_map.timeDependent=cms.bool(True) \""},
```

Comment:
If the `timeDependent=True` is needed only for `PreMixingModule`, why the customization is needed for RECO jobs?

(ok, I do see `ecal_sim_parameter_map` getting propagated into validation
https://github.com/cms-sw/cmssw/blob/22319db280b47d7712fc375133d61be78283fe19/Validation/EcalDigis/python/ecalMixingModuleValidation_cfi.py#L6-L7
but it should be affected by the same problem of the customization not being propagated there)

In a longer term I believe a modifier (in `Configuration.ProcessModifiers`) would be more practical way for these customizations than `--customise_commands` (which would need #22492 to be fixed).

Reply:
True, it should not be needed later in the processing workflow.
I am going to remove it.
About the "ecalMixingModuleValidation_cfi.py" I don't know.

Reply:
> In a longer term I believe a modifier (in `Configuration.ProcessModifiers`) would be more practical way for these customizations than `--customise_commands` (which would need #22492 to be fixed).

In the longer term, the conditions customizations would be removed because the conditions would be delivered by an actual global tag. At this point, however, AlCa would prefer to avoid introducing a new queue.